scripts/bash/run_conformance.sh

```diff
@@ -18,7 +18,7 @@ SRC_DIR=`dirname "$0"`
 source ${SRC_DIR}/enceladus_env.sh
 
 export CLASS=${CONF_CLASS}
-export JAR=${SPARK_JOBS_JAR}
+export JAR=${SPARK_JOBS_JAR:=$SPARK_JOBS_JAR_OVERRIDE}
```

Comment:
for it to be an override you would have to do
```suggestion
export JAR=${SPARK_JOBS_JAR_OVERRIDE:=$SPARK_JOBS_JAR}
```

Reply:
`--jar` allows you to override the jar location from the command line. There is also `--class`, but there is a bug in the script and it doest it wrong. You can fix `--class` if you want.

Reply:
How about the user having to specify `SPARK_JOBS_JAR_OVERRIDE` so it is a bit more explicit?

Reply:
Left a comment: https://github.com/AbsaOSS/enceladus/issues/1400#issuecomment-659971243